GSA-TTS/FAC
# New Submission Checklist

# Submission Checklist Updates (First Real Implementation)

Issue: #1560 
Figma: [Submission checklist](https://www.figma.com/file/lnkXJTfKzD3YBhiJZ2QFEv/FAC-final?node-id=1%3A31587&mode=dev)

## Changes:
1. The big change is the new submission checklist HTML. 
    * The old checklist (`submission-progress.html`) was moved and renamed to `submission-checklist.html`. 
    * It is in its own subfolder, `/audit/templates/audit/submission_checklist`, with a few helper components.
    * It's a little long. I chose to use larger if/else blocks to avoid complexity that will make things harder to change in the future. 
2. Some `main.scss` changes to help bring things closer to Figma in the future. These are utility changes, so only components that make use of the relevant utility are changed. This includes: 
    * Making `font-[type]-md` 18 point, instead of 17. 
    * Making `font-[type]-lg` 36 point, instead of 32.
    * Making the value for `success-lighter` much lighter (equivalent to 0.1 opacity on a white background), for use as a background color. `#56B833` -> `#eef8eb`

## How to test:
1. Switch to this branch and run normally.
2. Make a new submission (view an old one), and check out the new checklist.
3. Play around with submitting things and viewing how the checklist changes. You can also change things in your DB to skip steps. 

## Screenshots:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/75599221-f3ec-44b9-a753-0a2e75322b73)
